app/pages/adjustments/profitOrLoss/GoodsAndServicesAmountPage.scala

```diff
@@ -28,5 +28,8 @@ case object GoodsAndServicesAmountPage extends AdjustmentsBasePage[BigDecimal] {
     routes.CheckNetProfitLossController.onPageLoad(taxYear, businessId)
 
   override def hasAllFurtherAnswers(businessId: BusinessId, userAnswers: UserAnswers): Boolean =
-    false // TODO should it go to CYA or to the summary page CheckNetProfitLossController
+    userAnswers.get(this, businessId).isDefined && CurrentYearLossesPage.hasAllFurtherAnswers(
```

Comment:
Bobby confirmed that we always go straight back to CYA page, don't need to worry about the info summary page